camel/models/openai_compatible_model.py

```diff
@@ -49,15 +50,14 @@ class OpenAICompatibleModel(BaseModelBackend):
     def __init__(
         self,
         model_type: ModelType,
-        model_config_dict: Dict[str, Any],
         api_key: str,
         url: str,
+        model_config_dict: Optional[Dict[str, Any]] = None,
```

Comment:
I think it matters to keep the param list follow the same order in all the different types of backend model. Maybe we can adjust the order of params in other backend model classes?